python/ray/_private/ray_logging/formatters.py

```diff
@@ -10,16 +11,26 @@ def generate_record_format_attrs(
     exclude_standard_attrs,
 ) -> dict:
     record_format_attrs = {}
+
+    # Python logging module starts to use `time.time_ns()` to generate `created`
+    # from Python 3.13 to avoid the precision loss caused by the float type. Here,
+    # we generate the `created` for the LogRecord to support older Python versions.
+    ct = time.time_ns()
+    record.created = ct / 1e9
```

Comment:
Actually changing `created` inside formatter is probably not a good idea: the same record with different formatters will see different timestamp.

Reply:
Maybe we should explore the `setLogRecordFactory`: https://stackoverflow.com/questions/31328300/python-logging-module-logging-timestamp-to-include-microsecond